content/patterns/alert/examples/alert.html

```diff
@@ -107,11 +106,11 @@ <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
               <th scope="row">
                 <code>aria-atomic=<q>true</q></code>
               </th>
-              <td>Implicit on <code>div</code></td>
+              <td><code>div</code> に暗黙的</td>
               <td>
                 <ul>
-                  <li>This does not have to be declared in the code because it is implicit in the alert role.</li>
-                  <li>Tells assistive technologies to use the entire content of the alert element as the alert message even if only a portion of it has changed.</li>
+                  <li>これは alert のロールにおいて暗黙的であるため、コードで宣言する必要はありません。</li>
+                  <li>アラート要素の内容の一部のみが変更された場合でも、アラートメッセージとしてアラート要素の全内容を使用するように、支援技術に指示します。</li>
```

Comment:
JISのルールに合わせると「のみ」→「だけ」らしいです。
原文では「element」は一回しか出現していないので、重ねて言わなくても良いのかも。

```suggestion
                  <li>アラートの要素の内容の一部だけが変更された場合でも、内容の全部をアラートメッセージとして使用するように、支援技術に指示します。</li>
```

(しかし「の」の連続は気になる……)